lib/irc-colors.js

```diff
@@ -21,7 +21,8 @@ const styles = {
   normal    : '\x0F',
   underline : '\x1F',
   bold      : '\x02',
-  italic    : '\x1D'
+  italic    : '\x1D',
+  inverse   : '\u0016'
```

Comment:
Would rather keep things consistent and use the `\x00` format
```suggestion
  inverse   : '\0x16'
```

Reply:
Sorry, supposed to be `\x16`

Reply:
Perfect :-)